Dockerfile

```diff
@@ -1,19 +1,53 @@
-FROM ubuntu:bionic
+# syntax=docker/dockerfile:experimental
 
-# Set up environment
-ENV LANG C.UTF-8
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:focal AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+COPY requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:10-slim AS yarn-dependencies
```

Comment:
`FROM node:12-slim AS yarn-dependencies`

We are updating all the projects to node12 which is the current LTS.